### DISCUSSION TITLE
my cargo build failed

### DISCUSSION THREAD

**Discussion by discipulus262** [bevyengine/bevy#8842] [Q&A]
I tried to build a project and it would not compile because the compiler  "failed to run custom build command for  'libudev-sys v0.1.4'" this has happened multiple times. I was wondering what was wrong and how to fix it

**Comment by NiklasEi** [comment]
You are most probably missing dependencies. From the missing package (`libudev-dev`) I guess you are on Linux. Please check https://github.com/bevyengine/bevy/blob/main/docs/linux_dependencies.md and make sure you have all dependencies installed.

**Reply by discipulus262** [reply]
thanks
